xml/System.Net.Security/SslStream.xml

```diff
@@ -4346,7 +4367,7 @@ This property gets the cipher suite that is going to be used in the communicatio
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets the name of the server the client is trying to connect to. That name is used for server certificate validation. It can be a DNS name or an IP address.</summary>
```

Comment:
```suggestion
        <summary>Gets the name of the server the client is trying to connect to. That name is used for server certificate validation. It can be a DNS name or an IP address.</summary>
```